client/gribiclient.go

```diff
@@ -799,7 +813,7 @@ func (c *Client) clearPendingOp(op *spb.AFTResult) (*OpResult, error) {
 	case spb.AFTResult_FIB_PROGRAMMED, spb.AFTResult_FIB_FAILED:
 		delete(c.qs.pendq.Ops, op.Id)
 	case spb.AFTResult_RIB_PROGRAMMED:
-		if c.state.SessParams.GetAckType() != spb.SessionParameters_RIB_AND_FIB_ACK {
+		if c.state.SessParams.GetAckType() != spb.SessionParameters_RIB_AND_FIB_ACK || c.state.SessParams.GetAckType() == spb.SessionParameters_RIB_AND_FIB_ACK && TreatRIBACKAsCompletedInFIBACKMode {
```

Comment:
Is it a best practice to use bracket here to increase clarity? 

```
(c.state.SessParams.GetAckType() != spb.SessionParameters_RIB_AND_FIB_ACK) || (c.state.SessParams.GetAckType() == spb.SessionParameters_RIB_AND_FIB_ACK && TreatRIBACKAsCompletedInFIBACKMode)
```

Reply:
If this makes it clearer, happy to add them :-)